### ISSUE TITLE
DEI/Inclusivity statement for  job posters and profiles

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/design-gc-digital-talent#34]
# ⌨ DEI Issue

Internal users have requested brief inclusivity statements that:

- advise candidates of how we provide equitable access during our staffing /recruitment processes for people who might otherwise be excluded or marginalized, such as those belonging to  equity-deserving groups.

- allow managers to adopt or adjust standard text on job posters to show how they are operationalizing the platform pledge to inclusivity on the ground with each recruitment process. 

## 🦋 Expected Behaviour

Users  understand how our recruitment processes integrate and use their self-declaration information if they provide it. 

## 🕵️ Details

User feedback from DTL colleagues in other directorates indicate that statement would help to make it clear why self-declaration data is being collected during any recruitment process. 

## 📐 Figma Design Files

 -  [Profile info](https://www.figma.com/file/UeTUDknj3s9ZgLCFF8eTcU/Candidates---Profile---Version-8?type=design&node-id=1-6308&mode=design&t=V6GUQEelHDqp4U1e-0)
- [ Pool creation and editing](https://www.figma.com/file/em5ArPr044YaIe1Vb1zCZz/Pools---Record-of-Decision?type=design&node-id=5115-12169&mode=design&t=UYj6EL35MSKZ7OWz-0)

# 🙋‍♀️ Proposed solutions 

### 📃 Profile Copy Suggestion 

**Diversity, Equity and Inclusion:** 

We are committed to building a skilled and diverse workforce that reflects the Canadians we serve. We welcome applicants regardless of their employment equity status, sexual orientation, and gender identity. Please note that we collect employment equity information in support of achieving a diverse workforce. Consideration and prioritization may be given to candidates self-identifying as belonging to one of the following Employment Equity groups: Aboriginal peoples, persons with disabilities, members of visible minorities, and women.

**Potential Location** 

 - DEI section of the application profile UI

## ✨ Potential Pool Operator Job Poster field

Use the textbox below to outline how your team will provide equitable access during this staffing /recruitment processes for those belonging to equity-deserving groups (e.g., Indigenous Peoples, people with disabilities, visible cultural minority groups, 2SLGBTQIA+, women).   

- Generic example text (**to potentially hide in an accordion**): 

Please note that we collect employment equity information in support of achieving a diverse workforce. Consideration and prioritization may also be given to candidates who indicate that they are a member of one  or more of the following Employment Equity groups: Indigenous peoples, persons with disabilities, members of visible cultural minority groups, and women. 

Text box limit Approx 100 characters.

# ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the issue was addressed.

- [ ] Candidates can read how DTL uses their DEI data in our recruitment efforts on the Job posters and on their profile form. 
- [ ] Pool operators can automatically include a generic statement or user-entered update.

**Comment by RM-1978** [comment]
Hi @gobyrne, is this a new consultation you would like done with the other DTL teams?

**Comment by gobyrne** [comment]
A broader DEI statement was being developed but is on hold. This is on hold until that unblocks or we get further direction from Lauren.

**Comment by gobyrne** [comment]
This issue is stale, closing
